keps/sig-scheduling/3022-min-domains-in-pod-topology-spread/README.md

```diff
@@ -292,13 +292,22 @@ rollout. Similarly, consider large clusters and how enablement/disablement
 will rollout across nodes.
 -->
 
+It shouldn't impact already running workloads. It's an opt-in feature,
```

Comment:
Will review tomorrow - in the meantime, if you're targeting beta, please update the kep.yaml and corresponding prr file.

Reply:
Sure. will update.